rosdep/python.yaml

```diff
@@ -6001,6 +6001,9 @@ python3-flask:
   nixos: [python3Packages.flask]
   rhel: [python3-flask]
   ubuntu: [python3-flask]
+python3-flask-restful:
```

Comment:
This is going to fail CI since it should be in alphabetical order.

Reply:
@waldgaenger friendly ping, mind making this alphabetical so the linter passes?